nusight2/src/client/components/vision/image.ts

```diff
@@ -10,12 +10,17 @@ interface BaseImage {
 
 export interface DataImage extends BaseImage {
   type: 'data'
-  data: IComputedValue<Uint8Array>
+  image: IComputedValue<Uint8Array>
 }
 
 export interface ElementImage extends BaseImage {
   type: 'element'
-  element: HTMLImageElement
+  image: HTMLImageElement
+}
+
+export interface BitmapImage extends BaseImage {
```

Comment:
Would be good to collapse `ElementImage` and `BitmapImage` into one type if we can

Reply:
I've unified them here: https://github.com/NUbots/NUbots/pull/945/commits/48885d43d9baf8a21328b04be62fda34bb1fb83c

`ElementOrBitmapName` was the best name I could come up with for the combined type. Happy for suggestions.